src/utils/config.ts

```diff
@@ -5,6 +5,8 @@ import fs from "fs"
 
 export const HOME = os.homedir()
 
+export type NoteAction = "remove" | "update"
```

Comment:
Can we make this an enum instead?
```suggestion
export enum NoteAction {
  Remove = "REMOVE",
  Update = "UPDATE"
```
Then to use it, we'll import the enum and it'd be `NoteAction.Remove`